### ISSUE TITLE
Disabled blocks are still shown

### ISSUE THREAD

**Issue by TomDeSmet** [benjamminf/craft-neo#99]
This was an issue with Matrix blocks too that got resolved in a previous Craft update.
Maybe you could update Neo to do the same?

**Comment by benjamminf** [comment]
Are you talking about in Live Preview mode? I can see disabled Neo blocks are still showing in Live Preview, but they are hidden on the main site. I'll get this fixed for the next release.

**Comment by TomDeSmet** [comment]
Yes, sorry, I should of been more specific. It's indeed an issue in Live Preview.

Thanks for the quick reply!

**Comment by groe** [comment]
Any news regarding this?

If it's not fixed yet, is there some workaround to prevent disabled neo blocks from showing up in the live preview?

**Comment by groe** [comment]
As a workaround I have added this to all loops over neo fields:

```twig
{% for block in blocks %}
  {#
  the isLivePreview/block.status-check can be removed once this has been fixed:
  https://github.com/benjamminf/craft-neo/issues/99
  #}
  {% if not (craft.request.isLivePreview and block.status == 'disabled') %}
    ...
  {% endif %}
{% endfor %}
```

Works for now.

**Comment by kblizeck** [comment]
@benjamminf Any word on this issue? I'm encountering it as well, and would like to avoid adding the fix from @groe above, if possible.
